docs/databases-overview.md

```diff
@@ -60,6 +63,7 @@ Shows which copy operations are implemented by default.
 | PostgreSQLDB | Yes          | Yes        | Yes
 | RedshiftDB   | Yes          | Yes        | Yes
 | BigQueryDB   | Yes          | Yes        |
+| DatabricksDB |
```

Comment:
should it be also added as a column then?
If I get the matrix intention right, it is showing the possibility to copy from and to the specific database?

Reply:
In case it means, it cannot so far read from the other DBs, add a NO?

Reply:
You are right, I forgot to add the column. This is now added. I used a dash instead of NO because I think this is reasier to overlook. Maybe a emoji ✔️ / 🚫 would be even better...

I did not add any COPY operation [here](https://github.com/mara/mara-db/blob/main/mara_db/shell.py#L563) because I did not spend time on it and I did not want to add something which is not proper tested. I personally do not have the need to copy data from Databricks to e.g. Postgres and copying into Databricks is currently not possible because `copy_from_stdin_command` is not implemented.

Reply:
To move data from other data sources to Databricks I export it to parquet files via [odbc2parquet](https://github.com/pacman82/odbc2parquet) (implemented via #56 / not published yet) and then create a unmanaged table in Databricks via `CREATE TABLE ... USING PARQUET LOCATION='<azure_storage_path>'`

Reply:
Nice hint about odbc2parquet. ;)
Generally, delta would in case outperform parquet. Especially, if you have many or huge files. ;) 
I thought maybe you would love to support Mara on top of Databricks as backend that performs the computations, you know? ;)

Reply:
@alexeyegorov Did you check out our Mara Slack Channel? I wrote there how you can run a mara pipeline inside a Databricks Notebook.

I have a draft PR which makes it possible to run a mara pipeline on another host through SSH or Docker/Docker Compose see https://github.com/mara/mara-pipelines/pull/69 . This might be helpful when you want to execute bash commands on a Databricks cluster node. Would love ❤️ to see contribution there.

Without that, you are only be able to use the power of a Databricks Cluster by running SQL/Spark commands against the  master node. But in most cases, I think people would be happy with the current Databricks support. Maybe I should write a How to run Mara on Databricks article some day…